.github/workflows/test-move-prover.yml

```diff
@@ -0,0 +1,39 @@
+name: Test_Move_Prover
+
+on:
+  repository_dispatch:
+    types: [new_PR]
+
+jobs:
+  install_and_test_Move_Prover:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Install Z3
+      uses: pavpanchekha/setup-z3@v1.2
+
+    - name: Install .NET Core SDK
+      run: |
+        sudo apt-get update
+        sudo apt-get install apt-transport-https
+        sudo apt-get update
+        sudo apt-get install dotnet-sdk-3.1
+    - name: Install Boogie
+      run: |
+        dotnet tool install --global Boogie
+    - name: Export the environment variable for Boogie
+      run: |
+        echo "::set-env name=BOOGIE_EXE::/home/runner/.dotnet/tools/boogie"
+    - name: Export the environment variable for Z3
+      run: |
+        echo "::set-env name=Z3_EXE::`which z3`"
+    - name: Checkout libra
+      uses: actions/checkout@v2
+      with:
+        path: 'libra'
+
+    - name: Test move-prover
+      working-directory: ./libra/language/move-prover/bytecode-to-boogie
+      timeout-minutes: 30 # normally, expected to be done within 10 mins.
+      run: cargo test
```

Comment:
what do you want to do at this point if the tests fail?

Right now if the tests fail, does the return value of `cargo test` reflect the error as well? If so, it will fail the job, which will only be visible in the action tab of this repo. This might be a good thing to start testing this github action, provided you periodically check the action tab here to see if any of them failed :D

If you want to post a comment on the issue, you'll have to replicate what I did in the other github action. But I think this is good enough for a first PR and you can work on that on a subsequent PR.

Reply:
Yes. When it fails, `cargo test` will return the error message. However, It seems that the return message of `cargo test` is too long because it builds first. Perhaps, I will split the job into two: one for build, and one for test.